### PR TITLE
Add issue and pull request templates

### PR DESCRIPTION
**Pull request by N0taN3rd**
This PR adds github issue and pr templates in order to help streamline the creation of both issues and pull requests.

### Issue Templates
When a user goes to open a new issue for this project they will have the choice to select from three templates or use the default blank slate issue. 
issuetemplates

### Pull Request Template
Designed for quick identification of the information pertaining to the particulars of the PR

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/replay_issue.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
+---
+name: Replay Issue
+about: Can not replay a web specific web page? Tell us about it!
```
Comment (ikreymer) [inline .github/ISSUE_TEMPLATE/replay_issue.md]: small typo 'web specific web'

File: .github/ISSUE_TEMPLATE/replay_issue.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+---
+## Page
+<!--- Tell us about the page, by at least giving us a URL! -->
+<!--- Here is the perfect place to include any details about how we failed to replay the page. -->
```
Comment (ikreymer) [inline .github/ISSUE_TEMPLATE/replay_issue.md]: Maybe change to 'Here is the perfect place to include any details about what is going wrong with the page compared to what is expected.

Comment (despens) [reply .github/ISSUE_TEMPLATE/replay_issue.md]: How about the classic "expected behavious" and "what actually happened"?

Comment (N0taN3rd) [reply .github/ISSUE_TEMPLATE/replay_issue.md]: Yes the classics are always better so per @despens comment will use "expected" vs "actual" here. 
Do we wish to keep the steps to reproduce number list example or just leave it blank?


### Changes to be applied
**Expected behavior**
`<!--- Tell us about the page and how it should replay -->`


**What actually happened**
`<!--- Tell us about how we failed to replay the page -->`

File: .github/ISSUE_TEMPLATE/replay_issue.md (hunk 1 above)
Comment (N0taN3rd) [reply .github/ISSUE_TEMPLATE/replay_issue.md]: Yup will fix

**Comment by ikreymer** [comment]
Looks good, though probably should rebase off of `master`, as they won't have effect unless added to master, so can just commit there on top of the 2.0.4 branch so we don't have to wait for next release to try them out!

**Pull request merged**